### PR TITLE
Add new help articles to sql migration dashboard

### PR DESCRIPTION
**Pull request by rchlkm**
This PR adds new articles to the help section of the dashboard.

image

### CODE REVIEW

File: extensions/sql-migration/src/dashboard/sqlServerDashboard.ts [hunk 1]
```diff
@@ -709,17 +712,35 @@ export class DashboardWidget {
...
-
-		linksContainer.addItems(links.map(l => this.createLink(view, l)), {
-			CSSStyles: {
-				'margin-bottom': '8px'
-			}
-		});
+		const links = [
+			{
+				title: localize('sql.migration.dashboard.help.link.migrateUsingADS', 'Migrate databases using Azure Data Studio'),
+				description: localize('sql.migration.dashboard.help.description.migrateUsingADS', 'The Azure SQL Migration extension for Azure Data Studio provides capabilities to assess, get right-sized Azure recommendations and migrate SQL Server database to Azure.'),
```
Comment (mokabiru) [inline extensions/sql-migration/src/dashboard/sqlServerDashboard.ts:718]:
```suggestion
				description: localize('sql.migration.dashboard.help.description.migrateUsingADS', 'The Azure SQL Migration extension for Azure Data Studio provides capabilities to assess, get right-sized Azure recommendations and migrate SQL Server databases to Azure.'),
```

Comment (mokabiru) [reply extensions/sql-migration/src/dashboard/sqlServerDashboard.ts]: @rchlkm Can you please fix a typo.

**Comment by mokabiru** [comment]
@rchlkm looks good. Added a comment for a typo

File: extensions/sql-migration/src/dashboard/sqlServerDashboard.ts [hunk 2]
```diff
@@ -709,17 +712,35 @@ export class DashboardWidget {
...
+				link: 'https://docs.microsoft.com/en-us/azure/dms/tutorial-sql-server-to-virtual-machine-online-ads'
+			},
+			{
+				title: localize('sql.migration.dashboard.help.link.dmsGuide', 'Azure Database Migration Guides'),
+				description: localize('sql.migration.dashboard.help.description.dmsGuide', 'A hub of migration articles that provides step-by-step guidance for migrating and modernizing your data assets in Azure.'),
+				link: 'https://docs.microsoft.com/en-us/data-migration/'
+			},
+			{
+				title: localize('sql.migration.dashboard.help.link.assessment', 'Assessment rules for Azure SQL Managed Instance'),
+				description: localize('sql.migration.dashboard.help.description.assessment', 'Assessment rules used to determine the feasibility of migrating your SQL Server instance to Azure SQL Managed Instance.'),
```
Comment (mokabiru) [inline extensions/sql-migration/src/dashboard/sqlServerDashboard.ts:738]: @rchlkm - Please remove this link and text

**Comment by mokabiru** [comment]
@rchlkm - code looks good. The screenshot above still has the typo. As long as the typo is good, please proceed.

File: extensions/sql-migration/src/dashboard/sqlServerDashboard.ts [hunk 3]
```diff
@@ -709,24 +712,38 @@ export class DashboardWidget {
...
-		linksContainer.addItem(videosContainer, {
-			CSSStyles: {
-				'margin-bottom': '8px'
-			}
-		});
+		const links = [
+			{
+				title: localize('sql.migration.dashboard.help.link.migrateUsingADS', 'Migrate databases using Azure Data Studio'),
+				description: localize('sql.migration.dashboard.help.description.migrateUsingADS', 'The Azure SQL Migration extension for Azure Data Studio provides capabilities to assess, get right-sized Azure recommendations and migrate SQL Server databases to Azure.'),
+				link: 'https://docs.microsoft.com/en-us/azure/dms/migration-using-azure-data-studio'
```
Comment (brian-harris) [inline extensions/sql-migration/src/dashboard/sqlServerDashboard.ts:719]: url links should not include the locale.
https://docs.microsoft.com/en-us/azure/dms/migration-using-azure-data-studio
should be
https://docs.microsoft.com/azure/dms/migration-using-azure-data-studio
they will get redirected to the proper page from here

Comment (brian-harris) [reply extensions/sql-migration/src/dashboard/sqlServerDashboard.ts:719]: same for other urls below

Comment (mokabiru) [reply extensions/sql-migration/src/dashboard/sqlServerDashboard.ts:719]: @brian-harris good catch, agreed we should remove the locale.

Comment (rchlkm) [reply extensions/sql-migration/src/dashboard/sqlServerDashboard.ts]: thanks Brian, I've removed the locales in the all the links

**Pull request merged**